### PR TITLE
Beautify README and add instructions for lazy.nvim

### PR DESCRIPTION
**Pull request by warbacon**
This PR is focused on transferring the changes made to the `README.md` file from my fork _(now deprecated)_.

The updates include a more eye-catching header, added emojis to highlight sections, slight modifications to the installation instructions, and other minor improvements.

I am open to feedback and ready to make any changes as needed.

**P.S.** `packer.nvim` has been archived, and the most common way to install plugins in Neovim now is by using `lazy.nvim`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,62 @@
...
+### [lazy.nvim](https://lazy.folke.io/)
 
-```vim
-Plug 'fladson/vim-kitty'
+> [!IMPORTANT]
+> Lazy.nvim only works in Neovim and is the recommended method for it.
+
+```lua
+{
+    "warbacon/vim-kitty",
```
Comment (fladson) [inline README.md]: oops 😁

File: README.md [hunk 2]
```diff
@@ -1,46 +1,62 @@
...
-### packer
+### [vim-plug](https://github.com/junegunn/vim-plug)
+
+> [!NOTE]
+> Recommended method for Vim. Also works in Neovim but this is _Vimscript_.
 
 ```vim
-use "fladson/vim-kitty"
+" You can select a tagged version
+Plug 'warbacon/vim-kitty', { 'tag': '*' }
```
Comment (fladson) [inline README.md]: Same here.

**Review by fladson — CHANGES_REQUESTED** [review]
Thanks for the improvements, looking great, just need to update the ref to match the repo.

**Comment by warbacon** [comment]
Done. Copy and paste problems 🫠.

**Review by fladson — APPROVED** [review]

**Pull request merged**